support-policy.md

```diff
@@ -0,0 +1,34 @@
+Support and Stability policy
+============================
+
+With regards to current and future releases the OpenSSL project has adopted the
+following policy:
+
+* Version 3.0 will be supported until 2023-09-07.
```

Comment:
The specifics shouldn't be part of the policy.  We should specify the minimum duration of support but not mention version numbers.

We could also point to a web page with the specifics.